### DISCUSSION TITLE
Contributing to documentation

### DISCUSSION THREAD

**Discussion by kflanagan** [advplyr/audiobookshelf#1448] [General]
Greetings,  I just found this app and it really fits a need that I have.  
I found some significant issues with the installation that would really have been smaller if documentation had called out a few key things.
Thankfully I got help from MediaCowboy in the Discord.

I'd be happy to contribute while the pain is fresh /er lessons are fresh.  I don't see the installation guide here in the repo though. 

We have recently been moving towards documentation as code at work, so I'm learning markdown. It's not as fancy as the official docs page, but it's really easy. 

No matter what, I'll at least write up some notes and drop them here.

**Comment by advplyr** [comment]
That would be great. The docs page are all markdown files that you can find here https://github.com/advplyr/audiobookshelf-web/tree/master/content/docs/install

I linked the ones related to install but you can go back a directory and find all the rest. Let me know if you have questions and thanks for offering to contribute

**Reply by kflanagan** [reply]
No dev branch?  I can do a commit now, it's pretty simple, just a little bit of stuff in the Docker page.

**Reply by advplyr** [reply]
Nope
